go.mod

```diff
@@ -409,7 +407,7 @@ replace (
 	github.com/spf13/afero => github.com/spf13/afero v1.2.2
 	github.com/spf13/cast => github.com/spf13/cast v1.3.0
 	github.com/spf13/cobra => github.com/spf13/cobra v1.1.1
-	github.com/spf13/jwalterweatherman => github.com/spf13/jwalterweatherman v1.1.0
+	github.com/spf13/jwalterweatherman => github.com/spf13/jwalterweatherman v1.0.0
```

Comment:
is good to downgrade?

Reply:
it's a transitive dependency and we don't carry the code in our vendor/ directory, so we should be fine

```
[dims@dims-a02 11:46] ~/go/src/k8s.io/kubernetes ⟩ go mod graph | grep github.com/spf13/jwalterweatherman
github.com/spf13/viper@v1.7.0 github.com/spf13/jwalterweatherman@v1.0.0
[dims@dims-a02 11:46] ~/go/src/k8s.io/kubernetes ⟩ go mod graph | grep github.com/spf13/viper | grep -v "^github.com/spf13/viper@v1.7.0 "
github.com/spf13/cobra@v1.1.1 github.com/spf13/viper@v1.7.0
github.com/spf13/cobra@v1.0.0 github.com/spf13/viper@v1.7.0
github.com/spf13/cobra@v0.0.3 github.com/spf13/viper@v1.7.0
```